man/smoothIntensity-methods.Rd

```diff
@@ -26,7 +27,12 @@ This method smoothes the intensity values of a
     selected \code{method}.}
   \item{\dots}{arguments to be passed to \code{method}. \code{SavitzkyGolay} has
     an additional \code{polynomialOrder} argument (default: \code{3}) to control
-    the order of the filter. Unused for \code{MovingAverage}}
+    the order of the filter. Unused for \code{MovingAverage}. 
```

Comment:
`\dots` isn't "unused" in MovingAverage anymore. This sentence could be removed.

Reply:
Sorry.. missed that one. Done.